modules/tcp/scanner.go

```diff
@@ -0,0 +1,234 @@
+// Package tcp contains the module implementation for TCP
+//
+// The module performs a complete TCP handshake to then send TCP packages. The default port is 443
+package tcp
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2"
+)
+
+type Flags struct {
+	zgrab2.BaseFlags
+	zgrab2.TLSFlags
+
+	Protocol  string `long:"protocol" default:"\\n" description:"Send an initiation request of the VPN protocol specified here." `
```

Comment:
What are possible options for Protocol?

Reply:
This can be either "pptp" or "openvpn". I added some information on the session parameters and on how to use the module in the README.md.

Reply:
I suggest to put these two possible values in the description here as well.

Reply:
Put the two possible values in the description of the `Protocol` flag as well.

Reply:
I did that in the latest version, the commit you are referencing here is outdated.

Reply:
Right, then it looks good to me :+1: